client/allocrunner/networking_bridge_linux.go

```diff
@@ -156,69 +165,61 @@ func (b *bridgeNetworkConfigurator) Teardown(ctx context.Context, alloc *structs
 	return b.cni.Teardown(ctx, alloc, spec)
 }
 
-func buildNomadBridgeNetConfig(b bridgeNetworkConfigurator, withConsulCNI bool) []byte {
-	var consulCNI string
-	if withConsulCNI {
-		consulCNI = consulCNIBlock
+func buildNomadBridgeNetConfig(b bridgeNetworkConfigurator, withConsulCNI bool) ([]byte, error) {
+	// Update website/content/docs/networking/cni.mdx when the bridge configuration
+	// is modified. If CNI plugins are added or versions need to be updated for new
+	// fields, add a new constraint to nomad/job_endpoint_hooks.go
+	var ranges []Range
+	allocRange := Range{Subnet: b.allocSubnet}
+	ranges = append(ranges, allocRange)
+
+	var routes []Route
+	allocRoute := Route{Dst: "0.0.0.0/0"}
+	routes = append(routes, allocRoute)
+
+	loop := Loopback{Type: "loopback"}
+
+	NewFirewall := Firewall{
+		Type:                   "firewall",
+		Backend:                "iptables",
+		IptablesAdminChainName: cniAdminChainName,
```

Comment:
to me this setup is already paying off -- we can just use the var name in the place that it's needed, without counting args in fmt.Sprintf